### PR TITLE
Cleanup adaptive scalar unit test

### LINKED ISSUE #2022: Cleanup adaptivescalar unit tests
Part of #2000

### PR DESCRIPTION
**Pull request by oxtopus**
Fixes #2022

### CODE REVIEW

File: tests/unit/nupic/encoders/adaptivescalar_test.py [hunk 1]
```diff
@@ -29,104 +29,117 @@
...
     """Non-periodic encoder, min and max specified"""
 
-    self.assertTrue((self._l.encode(1) == numpy.array([1, 1, 1, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0],
-                                       dtype=defaultDtype)).all())
-    self.assertTrue((self._l.encode(2) == numpy.array([0, 1, 1, 1, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0],
-                                       dtype=defaultDtype)).all())
-    self.assertTrue((self._l.encode(10) == numpy.array([0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 1, 1, 1, 1],
-                                        dtype=defaultDtype)).all())
+    self.assertTrue(numpy.array_equal(
+      self._l.encode(1),
```
Comment (david-ragazzi) [inline tests/unit/nupic/encoders/adaptivescalar_test.py]: @oxtopus What the advantage of use `numpy.array_equal` over direct comparison?

**Comment by david-ragazzi** [comment]
:+1: But need wait Travis gets green on OS X (I restarted the build).

File: tests/unit/nupic/encoders/adaptivescalar_test.py (hunk 1 above)
Comment (oxtopus) [inline tests/unit/nupic/encoders/adaptivescalar_test.py]: `==` is element-wise, so you get another list back, which means you have to call `all()` to check if they are all equal element-wise.  `numpy.array_equal(A, B)` is more readable than `(A == B).all()`

Comment (david-ragazzi) [inline tests/unit/nupic/encoders/adaptivescalar_test.py]: Hmm.. ok.. Thanks!

**Comment by david-ragazzi** [comment]
:+1:

**Pull request merged**